index.html

```diff
@@ -1055,19 +1057,104 @@ <h2>
-              includes the various attributes and internal slots of
-              <var>request</var> (some MAY be excluded for privacy reasons
-              where appropriate).
+              element in the <var>paymentMethod</var> tuple and
+              <var>modifiers</var>. Optionally, the user agent SHOULD send the
+              appropriate data from <var>request</var> to the user-selected
+              <a>payment handler</a> in order to guide the user through the
+              payment process. This includes the various attributes and other
+              internal slots of <var>request</var> (some MAY be excluded for
+              privacy reasons where appropriate).
+            </p>
+            <p>
+              Handling of multiple applicable modifiers in the
+              <a>[[\serializedModifierData]]</a> internal slot is <a>payment
+              handler</a> specific and beyond the scope of this specification.
+              Nevertheless, it is RECOMMENDED that <a>payment handlers</a> use
+              a "last one wins" approach with items in the
+              <a>[[\serializedModifierData]]</a> list: that is to say, an item
+              at the end of the list always takes precedence over any item at
+              the beginning of the list (see example below).
             </p>
+            <aside class="example" title=
+            "Handling of multiple applicable modifiers">
+              <p>
+                This example uses the <a data-cite=
+                "payment-method-basic-card">Basic-Card</a> payment method to
+                demonstrate precedence order of
+                <a>[[\serializedModifierData]]</a>. The first modifier applies
+                equally to all cards, irrespective of network. The second
+                modifier applies specifically to cards on the "visa" network.
+              </p>
+              <pre class="js">
+// All cards incur a $3.00 processing fee.
+const cardFee = {
+  label: "Card processing fee",
+  amount: { currency: "USD", value: "3.00" },
+};
+
+// But visa card incurs a $1.00 processing fee.
+const visaFee = {
+  label: "Visa processing fee",
+  amount: { currency: "USD", value: "1.00" },
+};
+
+// Modifiers apply when the user chooses to pay with
+// a credit card.
+const modifiers = [
+  // Applies to all cards...
+  {
+    additionalDisplayItems: [cardFee],
+    supportedMethods: "basic-card",
+    total: {
+      label: "Total due",
+      amount: { currency: "USD", value: "53.00" },
+    },
+    data: {
+      supportedNetworks: [], // All networks
+    },
+  },
+  // Applies only to visa cards...
+  {
+    additionalDisplayItems: [visaFee],
+    supportedMethods: "basic-card",
+    total: {
+      label: "Total due",
+      amount: { currency: "USD", value: "51.00" },
+    },
+    data: {
+      supportedNetworks: ["visa"] // Visa network only
+    },
+  },
+];
+Object.assign(details, { modifiers });
```

Comment:
Not sure where `details` is declared, but just `details.modifiers = modifiers` is simpler.